.prettierrc.yaml

```diff
@@ -1,2 +1,5 @@
-printWidth: 80
+semi: false
+singleQuote: true
```

Comment:
Do we prefer single quotes or should we switch to Prettier defaults - double quotes?

https://prettier.io/docs/options#quotes

Reply:
My preference: use single quotes